doc/quickstart.rst

```diff
@@ -526,20 +526,76 @@ User scripts
 Your experiment commands can be sequenced into scripts,
 to be executed one after the other as you desire.  To set this up, write a sequence of commands into a text file,
 save it with the extension ``.py`` in the ``userScripts`` directory with a memorable name, like ``myNightShiftScript.py``.
-Double and triple check your script, then when you are ready to execute it, in ``ipython`` session type:
 
+Here is an example for a script that runs two grid scans with different mesh sizes.
```

Comment:
this should not be in quickstart, it is way too complicated, and using bluesky built-in plans.  Let's leave the quickstart as it was and we should discuss what we want to do in the way of scripts that users can use.

We can move this level of scripting to v0.6, and maybe old-style scripting that works but is not so robust, but easier to understand,  to v0.5patch.

Of course users can write scripts like this if they read bluesky documentation, but I don't want them seeing this in our xpdAcq documentation.